src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>Why are the IDs being converted to floats?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>They are being converted to int, not floats.</p>

Reply:
<p>Sorry - I meant integers, not floats. These should be classification variables, right? Why convert to ints? (It may not matter under the hood, just curious if this is impacting the model.)</p>

Reply:
<p>pickup_location_id(start_zone_id) and dropoff_location_id(end_zone_id) are definitely has no meaning themselves, but they are used in the feature engineering part to get the centroid of each zone and converted into the longitude and latitude points.</p>